.github/workflows/linux_openvino_ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Linux OpenVINO CI
+
+on:
+  push:
+    branches: [ main, 'rel-*' ]
+  pull_request:
+    branches: [ main, 'rel-*' ]
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  packages: write # Needed if the reusable workflow pushes images
+  attestations: write # Optional: for artifact attestations if enabled
+  id-token: write # Optional: may be needed for OIDC authentication (e.g., ACR)
+
+jobs:
+  build_test_openvino:
+    name: Build and Test OpenVINO EP (Ubuntu 22.04, Py3.10)
```

Comment:
The Windows OpenVINO CI Pipeline uses Python 3.12, while we are using Python 3.10 here. Is this intentional? Should we consider updating to the same version for consistency?

Reply:
Thanks for the suggestion. I update the setting. Please review it again.